<launch>
    <!--加载车辆模型-->
    <include file="$(find my_urdf)/launch/display.launch"/>
    <!--加载底层驱动-->
    <!--此步发布Imu,gps,Imu_adc,robot_adc以及transform-->
    <node pkg="my_pkg" name ="car_driver" type="car_driver" output="screen" respawn="true">
    <rosparam file="$(find my_pkg)/config/robot_param.yaml" command="load"/></node>
    <include file="$(find robot_localization)/launch/ekf_template.launch"/>
    <include file="$(find lslidar_driver)/launch/lslidar_serial.launch"/>
</launch>